CHANGES.md

```diff
@@ -60,8 +60,9 @@ OpenSSL 3.0
 
    *Richard Levitte*
 
- * The command line utilities ecparam and ec have been deprecated.  Instead
-   use the pkeyparam, pkey and genpkey programs.
+ * The command line utilities ecparam and ec have been modified to use PKEY
+   APIs in anticipation of their later deprecation.  Instead use the pkeyparam,
```

Comment:
Why do I not see any changes to ecparam.c and ec.c in this PR?

Reply:
They didn't need any -- they used EVP already.

I'll remove this comment.

Reply:
And done.